app/src/androidTest/java/com/github/swent/echo/compose/components/TopBarTest.kt

```diff
@@ -1,104 +1,3 @@
-
-    private var searchClicked = 0
-    private lateinit var navActions: NavigationActions
-
-    @Before
-    fun setUp() {
-        searchClicked = 0
-        navActions = mockk(relaxed = true)
-        composeTestRule.setContent { TopBar(navActions, { searchClicked++ }) }
-    }
-
-    @Test
-    fun shouldShowEcho() {
-        composeTestRule.onNodeWithText("Echo").assertExists()
-    }
-
-    @Test
-    fun shouldCallOpenSearchCallbackWhenSearchButtonClicked() {
-        composeTestRule.onNodeWithTag("list_map_mode_button").performClick()
-        assertThat(searchClicked, equalTo(1))
-    }
-
-    @Test
-    fun shouldOpenHamburgerMenuWhenMenuButtonClicked() {
-        composeTestRule.onNodeWithTag("menu_button").performClick()
-        composeTestRule.onNodeWithTag("hamburger_menu").assertExists()
-    }
-
-    @Test
-    fun shouldShowProfileBoxWhenMenuButtonClicked() {
-        composeTestRule.onNodeWithTag("menu_button").performClick()
-        composeTestRule.onNodeWithTag("profile_box").assertExists()
-    }
-
-    @Test
-    fun shouldShowProfileSheetWhenMenuButtonClicked() {
-        composeTestRule.onNodeWithTag("menu_button").performClick()
-        composeTestRule.onNodeWithTag("profile_sheet").assertExists()
-    }
-
-    @Test
-    fun shouldShowProfilePictureWhenMenuButtonClicked() {
-        composeTestRule.onNodeWithTag("menu_button").performClick()
-        composeTestRule.onNodeWithTag("profile_picture").assertExists()
-    }
-
-    @Test
-    fun shouldShowProfileInfoWhenMenuButtonClicked() {
-        composeTestRule.onNodeWithTag("menu_button").performClick()
-        composeTestRule.onNodeWithTag("profile_info").assertExists()
-    }
-
-    @Test
-    fun shouldShowProfileNameWhenMenuButtonClicked() {
-        composeTestRule.onNodeWithTag("menu_button").performClick()
-        composeTestRule.onNodeWithTag("profile_name").assertExists()
-    }
-
-    @Test
-    fun shouldShowProfileClassWhenMenuButtonClicked() {
-        composeTestRule.onNodeWithTag("menu_button").performClick()
-        composeTestRule.onNodeWithTag("profile_class").assertExists()
-    }
-
-    @Test
-    fun shouldShowDarkModeButtonWhenMenuButtonClicked() {
-        composeTestRule.onNodeWithTag("menu_button").performClick()
-        composeTestRule.onNodeWithTag("dark_mode_button").assertExists()
-    }
-
-    @Test
-    fun shouldShowAllItemsWhenMenuButtonClicked() {
-        composeTestRule.onNodeWithTag("menu_button").performClick()
-        for (i in 0..6) {
-            composeTestRule.onNodeWithTag("navigation_item_$i").assertExists()
-        }
-    }
-
-    @Test
-    fun shouldCloseHamburgerMenuWhenNavigationItemClicked() {
-        composeTestRule.onNodeWithTag("menu_button").performClick()
-        for (i in 0..6) {
-            composeTestRule.onNodeWithTag("navigation_item_$i").performClick()
-            composeTestRule.onNodeWithTag("top_bar").assertExists()
-        }
-    }
-}
+class TopBarTest {}
```

Comment:
It might be a good idea to completely remove this file. If we need it in the future, it'll always be possible to find it in the git history.

Reply:
yes i think we can remove it